Firebase/Messaging/FIRMessagingPendingTopicsList.m

```diff
@@ -65,10 +68,7 @@ - (instancetype)initWithCoder:(NSCoder *)aDecoder {
   }
 
   if (self = [self initWithAction:action]) {
-    NSSet *topics = [aDecoder decodeObjectForKey:kPendingTopicBatchTopicsKey];
-    if ([topics isKindOfClass:[NSSet class]]) {
-      _topics = [topics mutableCopy];
-    }
+    _topics = [aDecoder decodeObjectOfClass:[NSSet class] forKey:kPendingTopicBatchTopicsKey];
```

Comment:
I wonder if it's working correctly without specifying the all classes available for decoding with [unarchivedObjectOfClasses:fromData:error:](https://developer.apple.com/documentation/foundation/nskeyedunarchiver/2962885-unarchivedobjectofclasses?language=objc)

Reply:
what do you mean all the classes? isn't _topics are defined as NSSet only?

Reply:
AFAIK the classes of the objects stored in `NSSet` also must be specified otherwise decoding will fail.